### ISSUE TITLE
Update DB to include information for securely storing passwords

### ISSUE THREAD

**Issue by jslopezgithub** [jslopezgithub/cnca#21]
Please refer to #20

**Comment by aliskhanoff** [comment]
@jslopezgithub Is not clear.

**Comment by jslopezgithub** [comment]
Did you read the link in #20?

**Comment by aliskhanoff** [comment]
Yes, i readed it and i know about encryption. Should i store HMAC key generated for each user into database? 
![HMAC](https://user-images.githubusercontent.com/41637477/70883644-efcd6f80-1fd3-11ea-9421-04a66ca464ee.png)

**Comment by jslopezgithub** [comment]
Based on the article, if you are to use password stretching what
information would you need to store in the database?


Il lun 16 dic 2019, 02:16 Ramazan aka 0xBADC0DED <notifications@github.com>
ha scritto:
